shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -55,4 +55,6 @@ FLUTTER_EXPORT
 
 - (void)handleMethodCall:(nonnull FlutterMethodCall*)call result:(nonnull FlutterResult)result;
 
+- (nonnull NSPasteboard*)getPasteboard;
```

Comment:
Why are you adding new public API to FVC to implement an internal channel? Unless there's a specific need for this to be public, it should be in the private header.

Reply:
I now have this as a property like the [comment](https://github.com/flutter/engine/pull/20531/files#r483133450) above recommends.  Let me know if how I did the property and getter doesn't look right. I just need to be able to mock it in the tests.

Excuse my lack of Objective C knowledge!